packages/frontend/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,44 @@
+import { useState } from "react";
+import {
+  Wrapper,
+  ImageContainer,
+  RecipeName,
+  BottomPanel,
+  RatingWrapper,
+  Rating,
+  StarIcon,
+  HeartIcon,
+  Author,
+} from "../Card/Card.styled";
+import starIcon from "src/assets/images/star.svg";
+import heartIcon from "src/assets/images/heart.svg";
+import { Routes } from "src/enums/routes.enum";
+import { CardProps } from "./Card.types";
+
+export const Card: React.FC<CardProps> = ({ id, photoUrl, title, rating, total, author }) => {
+  const [isFilled, setIsFilled] = useState(false);
+
+  const toggleFill = (e: React.MouseEvent<SVGElement, MouseEvent>) => {
```

Comment:
Ta funckja jak i isFilled powinna w przyszłości wpadać w propsach jak zostanie dorobione dodawanie do polubionych.   
Niepotrzebnie dla każdej karty tworzony jest stan i ta funckja.  
W przyszłości i tak toggleFill będzie wykonywało  i tak strzał do api.  
Zostaw tutaj proszę komentarz, żeby zostało to wyniesione w przyszłości do komponentu wyżej.

Reply:
zdecydowanie tak